google/cloud/storage/internal/create_object_acl_request_test.cc

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/storage/internal/create_object_acl_request.h"
+#include <gmock/gmock.h>
+
+namespace google {
+namespace cloud {
+namespace storage {
+inline namespace STORAGE_CLIENT_NS {
+namespace internal {
+namespace {
+
+TEST(CreateObjectAclRequestTest, Simple) {
+  CreateObjectAclRequest request("my-bucket", "my-object", "user-testuser",
+                                 "READER");
+  EXPECT_EQ("my-bucket", request.bucket_name());
+  EXPECT_EQ("my-object", request.object_name());
+  EXPECT_EQ("user-testuser", request.entity());
+  EXPECT_EQ("READER", request.role());
+}
+
+using ::testing::HasSubstr;
```

Comment:
Nit: Can you move this to the top, above all the test functions?  (I've got a CL coming soon to fix and organize a bunch of our using-declarations within tests, and would like them all to be in the same place)

Reply:
Done.